tfx/dsl/component/experimental/placeholders.py

```diff
@@ -54,3 +66,10 @@ class OutputUriPlaceholder(json_utils.Jsonable):
 
   def __init__(self, output_name: Text):
     self.output_name = output_name
+
+  def __eq__(self, other):
+    return isinstance(other, self.__class__) and \
```

Comment:
Try to avoid `\` line continuations, maybe the following if you need to adhere to the 80 character limit:

```python
return (isinstance(other, self.__class__) and
        self.output_name == other.output_name)
```